mne/preprocessing/xdawn.py

```diff
@@ -62,6 +69,12 @@ def _least_square_evoked(epochs_data, events, tmin, sfreq):
         An concatenated array of toeplitz matrix for each event type.
     """
 
+    if isinstance(solver, string_types):
+        if solver == "pinv":
+            from ..stats.regression import _pinv_solver as solver
+        elif solver == 'cholesky':  # noqa
```

Comment:
I don't *want* to test all options ... one is default for XDawn, the other for rERP. See:

>I'm not checking the pinv solver for linear_regression_raw nor the cho solver for xdawn

Any solution?